src/components/CircleCiBuilds/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Commit from '../Commit';
+import './styles.css';
+
+import { getCircleCiLatestBuilds, selectBuild } from '../../reducers/CircleCiReducer/actions';
+
+class CircleCiBuilds extends Component {
+  constructor(props) {
+    super();
+  };
+
+  componentDidMount() {
+    this.props.getCircleCiLatestBuilds();
+  }
+
+  selectBuild = (build, buildNo) => (e) => {
+    this.props.selectBuild({...build,  buildNo: buildNo});
+  }
+
+  render() {
+    return(
+      <div className='circle-ci-builds-page'>
+        <h2>Circle Ci</h2>
+        <div>
+          <label>Latest builds</label>
+        </div>
+        <div className='latest-builds'>
+          { 
+            this.props.latestBuilds.map((build, i) => 
+              <Link 
+                key={"commit-" + i} 
```

Comment:
the most proper key for this list would be what uniquely identifies the build:

vcs type + project + branch + build number